CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ set(VELOX_DEPENDENCY_SOURCE
       STRING
       "Default source for all dependencies with source builds enabled: AUTO SYSTEM BUNDLED."
 )
+set(VELOX_GFLAGS_TYPE "shared" CACHE STRING "Specify whether to find the gflags package as a shared or static package")
```

Comment:
Please use `option()` for options :)

Reply:
The option() command always creates a Boolean variable. However, find_package() requires you to pass a string constant, such as 'static' or 'shared.' Are you suggesting that we first define a Boolean variable using option(), then convert it into a 'static' or 'shared' string constant before passing it to find_package()？

Reply:
Ah, sorry in arrow we already have a helper function for string options so I was confused. In that case i think this is ok for now.